### PR TITLE
docs(readme): Add breaking changes notes

### PR DESCRIPTION
**Pull request by JoseLion**
This PR will be merged as soon as we release the new versions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,42 +18,66 @@ It also allows you to implement a highly customizable tour feature with an aweso
...
 ```bash
-$ yarn add @stackbuilders/react-native-spotlight-tour
+$ yarn add react-native-spotlight-tour
 ```
 
+## 🚨 Breaking changes: v2 to v3
+
+This major update brings a few fixes, some great new features, and some breaking changes. These are some highlight you'll need to consider while upgraging from v2 to v3:
+
+- The package has been rename from `@stackbuilders/react-native-spotlight-tour` to just `react-native-spotlight-tour`
```
Comment (ChristianSama) [inline README.md:52]:
```suggestion
- The package has been renamed from `@stackbuilders/react-native-spotlight-tour` to just `react-native-spotlight-tour`
```

**Review by ChristianSama — APPROVED** [review]
LGTM! Just found a small typo.

**Comment by JoseLion** [comment]
Awesome, thank you @ChristianSama! 🎉

File: README.md [hunk 2]
```diff
@@ -18,42 +18,67 @@ It also allows you to implement a highly customizable tour feature with an aweso
...
+- The package has been renamed from `@stackbuilders/react-native-spotlight-tour` to just `react-native-spotlight-tour`
+  - Dont worry, this library is still developed and maintained by the [Stack Builders Inc.](https://www.stackbuilders.com/) team!
+  - Remove the former package from your dependencies and use the command described in the [Install section](#install)
+  - Rename any import from the previous name to use just `react-native-spotlight-tour` instead
+- Tooltip positioning was refactored
+  - Props related to the tooltip position were removed from `SpotlightTourProvider` and the `TourStep` object.
+    - Both `Align` and `Position` enums were removed
+    - Both `alignTo` and `position` props were removed
+  - We now delegate the positioning to [FloatingUI](https://floating-ui.com/), so you can use the `floatingProps` prop to configure its global behavior or granularly on each step.
+  - Middleware functions are re-xported from `@floating-ui/react-native` so you can directly import them from `react-native-spotlight-tour`.
```
Comment (well1791) [inline README.md:62]:
```suggestion
  - Middleware functions are re-exported from `@floating-ui/react-native` to `react-native-spotlight-tour`.
```

**Review by well1791 — APPROVED** [review]
@JoseLion LGTM just one detail, not a change request though

File: README.md (hunk 2 above)
Comment (JoseLion) [reply README.md:62]: Nice catch! Thank you @well1791!

**Pull request merged**